website_no_crawler/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
you could update READM according to new specs https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst